chronograf/1.3/Dockerfile

```diff
@@ -1,8 +1,13 @@
 FROM buildpack-deps:jessie-curl
 
-RUN gpg \
-    --keyserver hkp://ha.pool.sks-keyservers.net \
-    --recv-keys 05CE15085FC09D18E99EFB22684A14CF2582E0C5
+RUN set -ex \
+  && for key in \
```

Comment:
Can you put the `&&` at the end of the line like the other areas and space this to match the other formatting?